src/Components/Validation/schema.js

```diff
@@ -0,0 +1,10 @@
+import * as Yup from 'yup'
+
```

Comment:
I suggest removing this line.

Reply:
Resolved on 8f30746300c932c00675d2cf20a67405767478ee